griptape/templates/engines/image_generation/prompt_image_generation.j2

```diff
@@ -0,0 +1,3 @@
+Description: """{{ description }}"""
+
+Generate an image based on the description provided.
```

Comment:
Let's sync up with @shhlife, @Amaru-Zeas, or @averyroche about what the best base prompt is.

Reply:
+1. I think we should leave the flowery specifics to the description field, but I can imagine reinforcing instructions like 'stick to the description' or optionally 'be creative', etc.